src/MultiTypeSymbolTable.chpl

```diff
@@ -320,19 +320,36 @@ module MultiTypeSymbolTable
         */
         proc info(name:string): string throws {
             var s: string;
-            if name == "__AllSymbols__" {
+            if tab.size == 0 {
+                s = "the symbol table is empty";
```

Comment:
I think we may want to return a different string here maybe something like `"__EMPTY_SYMBOLTABLE__"`
what do you think? I also worry about the interaction between and empty symboltable/registry and your `cleanup()` function in the test flow.

Reply:
@mhmerrill agreed, and perhaps an enum instead of a string?